.github/workflows/dependabot-auto-merge.yml

```diff
@@ -31,6 +31,8 @@ jobs:
         uses: dependabot/fetch-metadata@v1.3.3
         with:
           github-token: "${{ secrets.GITHUB_TOKEN }}"
+          # needed to avoid "Dependabot's commit signature is not verified, refusing to proceed" error when synchronizing manually the PR
+          skip-commit-verification: true
```

Comment:
actually this could be avoided by asking dependabot to rebase the PR -> not sure it's needed then, wdyt please?

Reply:
what do you mean with `when synchronizing manually the PR`?

I think it's because dependabot expect that all of its are commits are signed with their key, so for example if you rebase a branch with some dependabot commits in it, will break the signature

Reply:
I removed this after all because dependabot will not rebase a branch where someones else made some commits.
I mean clicking on the button to merge or rebase the PR when the PR shows the "This branch is out-of-date with the base branch” warning.